frontend/awx/resources/inventories/InventoryPage/InventoryHosts.tsx

```diff
@@ -35,6 +35,23 @@ export function InventoryHosts() {
       hostOptions.actions['POST'] &&
       params.inventory_type === 'inventory'
   );
+
+  let emptyStateTitle = '';
+  let emptyStateDescription = '';
+
+  if (params.inventory_type === 'inventory') {
+    emptyStateTitle = canCreateHost
+      ? t('There are currently no hosts added to this inventory.')
+      : t('You do not have permission to create a host.');
+
+    emptyStateDescription = canCreateHost
+      ? t('Please create a host by using the button below.')
+      : t('Please contact your organization administrator if there is an issue with your access.');
+  } else {
+    emptyStateTitle = t('No Hosts Found');
+    emptyStateDescription = t('Please add Hosts to populate this list');
+  }
+
```

Comment:
We need to decide whether we are capitalizing Hosts/Host here.  Currently we are doing a bit of both.

Reply:
I found more of those differences, some of them are corrected in my different PRs. I wanted to file an issue, lets communicate this in channel.